src/go/rpk/pkg/cli/registry/mode/reset.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package mode
+
+import (
+	"os"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/schemaregistry"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func resetCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	return &cobra.Command{
+		Use:   "reset [SUBJECT...]",
+		Short: "Reset schema registry mode",
+		Long: `Reset schema registry mode
+	
+This command deletes any subject modes and reverts to the global default. 
```

Comment:
Should we mention in the description what's the global default? Could we reference it in the code with a variable instead of hardcoding it?

Reply:
We don't have a way to know beforehand what's the Global Default, because it's a variable that users can change (even if it says it's a default haha)